internal/standby/standby.go

```diff
@@ -75,13 +76,61 @@ func (s *Service) RunMQTT(ctx context.Context) error {
 			}
 		})
 
+	s.subscribeToTopic(s.cfg.MQTT.ReadCommandTopic,
+		func(client mqtt.Client, msg mqtt.Message) {
+			s.logger.Debug(fmt.Sprintf("Received message: %s from topic: %s", msg.Payload(), msg.Topic()))
+			s.latestCommandReceived = time.Now()
+		})
+
 	return nil
 }
 
 func (s *Service) StopMQTT() {
 	s.MQTTClient.Disconnect(uint(1000))
 }
 
+func (s *Service) RunDetector(ctx context.Context) {
```

Comment:
how will this be invoked? a cronjob? a ticker?

Reply:
oh, I see, this has a ticker inside instead.
So you would be invoking this in a goroutine as a parent.